main/src/ca/uwaterloo/flix/language/phase/Simplifier.scala

```diff
@@ -244,7 +244,7 @@ object Simplifier {
         val e = visitExp(exp)
         SimplifiedAst.Expression.PutStaticField(field, e, tpe, simplifyPurity(pur), loc)
 
-      case TypedAst.Expression.NewObject(clazz, tpe, pur, eff, loc) =>
+      case TypedAst.Expression.NewObject(clazz, tpe, pur, eff, methods, loc) =>
```

Comment:
This field is ignored on purpose, right?

Reply:
For now, yes. I'll not ignore it once I add JvmMethod to SimplfiedAst.